src/test/c_api_tests.cpp

```diff
@@ -286,6 +288,151 @@ TEST_F(CAPIInference, TensorSetMovedBuffer) {
     ASSERT_EQ(tensor.setBuffer(std::move(buffer)), ovms::StatusCode::DOUBLE_BUFFER_SET);
 }
 
+TEST_F(CAPIInference, NoInputs) {
```

Comment:
Minor: fixture name might be misleading (CAPIInference) since no inference is happening in the test. The same goes for tests below (InferenceRequest/InferenceResponse)